README.md

```diff
@@ -1,52 +1,34 @@
 cloud-sdk-docker
 ================
 
-[`google/cloud-sdk`](https://index.docker.io/u/google/cloud-sdk/) is a [Docker](https://docker.io) image bundling all the components and dependencies
-of the [Google Cloud SDK](https://cloud.google.com/sdk/):
-
-- [App Engine SDK for Go](https://cloud.google.com/appengine/docs/go/)
-- [App Engine SDK for Java](https://cloud.google.com/appengine/docs/java/)
-- [App Engine SDK for Python](https://cloud.google.com/appengine/docs/python/) and [PHP](https://cloud.google.com/appengine/docs/php/)
-- [Big Query Command Line Tool](https://cloud.google.com/bigquery/bq-command-line-tool)
-- [Cloud DNS Admin Command Line Interface](https://cloud.google.com/dns/migrating-bind-zone-command-line)
-- [Cloud SQL Admin Command Line Interface](https://cloud.google.com/sql/docs/admin-api/)
-- [Cloud Storage Command Line Tool](https://cloud.google.com/storage/docs/gsutil)
-- [Compute Engine Command Line Tool](https://cloud.google.com/compute/docs/gcloud-compute/)
-- [Preview Command Line Tools](https://cloud.google.com/sdk/gcloud/reference/preview/)
```

Comment:
This component list was really out of date. We no longer have preview command line tools, and the app engine packages have been pulled out of gcloud. Changed it to just say it installs everything. (and actually install everything)